### PR TITLE
MS Teams - Adding ring-user command

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22329

## Description
Added ring-user command which rings a user on Teams.
This does not create an actual call but just a ring.

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by roysagi** [comment]
@Arsenikr @Zivsolo - FYI

File: Integrations/MicrosoftTeams/MicrosoftTeams.yml [hunk 1]
```diff
@@ -225,7 +225,33 @@ script:
...
+      required: true
+      secret: false
+    - default: false
+      description: The tenant ID for the Teams.
+      isArray: false
+      name: tenant_id
+      required: true
+      secret: false
+    - default: false
+      description: The callback URI for the call. The URI determins who will recieve
```
Comment (ronykoz) [inline Integrations/MicrosoftTeams/MicrosoftTeams.yml]: how will you get that as a user?

File: Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 2]
```diff
@@ -1495,6 +1495,74 @@ def messages() -> Response:
...
+
+    Notes:
+        This is a ring only! no media plays in case the generated call is answered.
+
+    Returns:
+        None.
+    """
+    bot_id = demisto.params().get('bot_id')
+    tenant_id = demisto.args().get('tenant_id')
+    callback_uri = demisto.args().get('callback_uri', 'https://callback.url')
```
Comment (ronykoz) [inline Integrations/MicrosoftTeams/MicrosoftTeams.py]: uri or url?
Add better explanation

**Comment by Itay4** [comment]
looks like we missed `microsoft-teams` prefix for most of the commands here
all of the commands but the following should have it: `send-notification`, `mirror-investigation` and `close-channel`
please add it to the and deprecate the others and also add it to the ring command you added

File: Integrations/MicrosoftTeams/MicrosoftTeams.yml (hunk 1 above)
Comment (roysagi) [reply Integrations/MicrosoftTeams/MicrosoftTeams.yml]: you are right there is no way I found to get this info. you can put a bot's callback_uri if you want the call to redirect to a different bot for some reason - I'll edit the description accordingly

File: Integrations/MicrosoftTeams/MicrosoftTeams.py (hunk 2 above)
Comment (roysagi) [reply Integrations/MicrosoftTeams/MicrosoftTeams.py]: it is called callback URI in the api but I can change to URL if you think it is more understandable

**Pull request closed without merging**